### ISSUE TITLE
Preferred citation format

### ISSUE THREAD

**Issue by Mike-Leo-Smith** [PearCoding/Ignis#23]
Hi, we are currently working on a rendering systems paper and would like to perform some comparisons with the project.

Do you have a preferred citation format for Ignis? So that we can properly cite your wonderful work.

**Comment by PearCoding** [comment]
Hi,
as there is no published paper for the renderer (yet), I would recommend citing the originel Rodent paper https://dl.acm.org/doi/10.1145/3306346.3322955, while refering to Ignis as a follow-up renderer (with a link or just a name).

However, Ignis does not "generate" renderers for a specific scene, like the Rodent renderer did, but applies JIT for shading only. This improves compile time and performance for large scenes, but decreases performance for small and simple scenes. The performance difference is most likely due to the register pressure and shader complexity.

In a nutshell, Ignis is based on a small subset of Rodents orignal code base, but has a completely different approach rendering a scene.

**Comment by Mike-Leo-Smith** [comment]
I see. Thank you so much for the detailed explanation!
